### ISSUE TITLE
[Feature] Logitech LED and Razer Chroma

### ISSUE THREAD

**Issue by Aerinx** [kershner/screenBloom#47]
Would be nice to add Razer and Logitech products to sync their colors with the screen colors as id they were additional bulbs.

**Comment by kershner** [comment]
I agree this would be awesome.  Probably pretty dang easy to implement as well, given that I believe there are public APIs for those products.

I don't currently have any of these products though so won't be doing any of the work myself.  I'm happy to help anyone that wants to take a stab at it thought.

**Comment by lucidlemon** [comment]
At best this app could merge with [Aurora](https://github.com/antonpup/Aurora) and give us the best of both worlds. They already have API access to Razer and Logitech Products. Plus you could do stuff like flashing red light when you are low hp on the hue.

**Comment by afeblot** [comment]
@kershner, what could help is if you set up a kind of bulb standard API, document it, and have such APIs work as plugins of your core program. Then we can easily develop such plugins independently and submit them to you so you just need to drop them in your plugin dir to make new bulb types available to be driven. 
If you set up such mechanism, I'd be happy to implement the driver for Xiaomi Yeelight bulbs built on top of https://github.com/skorokithakis/python-yeelight.
